x-pack/test/plugin_api_integration/plugins/event_log/server/init_routes.ts

```diff
@@ -42,6 +42,9 @@ export const logEventRoute = (router: IRouter, eventLogger: IEventLogger, logger
         await context.core.savedObjects.client.create('event_log_test', {}, { id });
         logger.info(`created saved object ${id}`);
       }
+      // mark now as start and end
```

Comment:
The only slightly odd thing here is that for the test where we search by end date, all of these events will have an end date - however, IRL, not all events have end dates.  We may need to do something in the future - like when searching via end date, add an additional filter to only return events with an end date - but let's wait to see how people use this, to see if something needs to be done.

Reply:
Yeah, agreed, though this is _sort by_ end rather than search by end.. semantics but it means that the behaviour is inline with how ES queries generally behave.